### ISSUE TITLE
Learn how to code

### ISSUE THREAD

**Issue by KSimonnet** [KSimonnet/helloWorld#2]
@jondubois: Hello, Comment ça va? Ayant changé récemment de taf, j'ai beaucoup plus de temps disponible et je me suis re-mis à apprendre à coder. J'ai commencé avec: 
- www.w3schools.com
- https://nodeschool.io/

Est-ce que tu aurais des sites, idéalement gratuit, à me recommander pour l'apprentissage de JS et Python?

**Comment by jondubois** [comment]
Ça fait longtemps que je n'ai pas étudié les bases donc je ne sais pas vraiment quel site te recommander.

Si tu décide de faire du front-end, je te recommande de commancer avec HTML, puis JavaScript, et après un framework comme VueJS: https://vuejs.org/

**Comment by KSimonnet** [comment]
oui, sur w3school j'ai appris le HTLML, CSS et là j'commence le JavaScript.

J'aimerais:
- faire du scraping à partir du JSON d'un site web de professional networking bien connu :-)
- héberger sur un site web des services de manip de data
À terme, créer mon propre Applicant Tracking System.

On m'a aussi recommander d'apprendre Python. Comme tout ça ressemble beaucoup à de l'ETL je vais pt^ regarder ça aussi plus en détails.

**Comment by KSimonnet** [comment]
@jondubois: Hello Jo'  5.5k stars ! Dis donc c'est que tu commences à être connu :-) je me rappel encore quand tu me disais qu'un projet qui obtient + de 10k stars est significatif.

Je me suis lancé dans la création d'une solution / startup et aurait aimé avoir ton avis / conseils.

Le concept est de rendre l'Exec Search accessible aux Recruteurs internes afin qu'une lean version soit appliquées aux postes - Senior / large volume. Un peu comme Canva a démocratisé  le UX/UI design.

J'ai rejoint l'incubateur Catalysr (https://catalysr.com.au) et vient d'être accepté à leur Accelerator Program. J'ai créé un dynamic sketch et un prototype en VBA sur ma local machine. On a un Demo Day mi-décembre où j'aurais aimé présenté un SaaS MVP.

Eske tu aurais un moment pour en parler (30 minutes) courant de semaine prochaine?
